GitCommands/Git/GitItemStatusConverter.cs

```diff
@@ -2,21 +2,33 @@
 {
     public static class GitItemStatusConverter
     {
+        public static readonly char AddedStatus = 'A';
+        public static readonly char CopiedStatus = 'C';
+        public static readonly char DeletedStatus = 'D';
+        public static readonly char ModifiedStatus = 'M';
+        public static readonly char IgnoredStatus = '!';
+        public static readonly char RenamedStatus = 'R';
+        public static readonly char SkippedStatus = 'S';
+        public static readonly char UnmergedStatus = 'U';
+        public static readonly char UnmodifiedStatus_v1 = ' ';
+        public static readonly char UnmodifiedStatus_v2 = '.';
+        public static readonly char UntrackedStatus = '?';
```

Comment:
I think these can be turned into consts...

Reply:
It will not be a practical problem, the const will not be used in other assemblies. But this is the case with all except the plugin interface.
I set this as `static readonly` to not have to make changes, will be happy to change to `const` with comments...